components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -285,6 +290,7 @@ func Plans(plans PlansConfig, provider internal.CloudProvider, includeAdditional
 		AzureLitePlanID: defaultServicePlan(AzureLitePlanID, AzureLitePlanName, plans, azureLiteSchema, AzureLiteSchema(azureLiteMachinesDisplay, azureLiteMachines, includeAdditionalParamsInSchema, true)),
 		FreemiumPlanID:  defaultServicePlan(FreemiumPlanID, FreemiumPlanName, plans, freemiumSchema, FreemiumSchema(provider, includeAdditionalParamsInSchema, true)),
 		TrialPlanID:     defaultServicePlan(TrialPlanID, TrialPlanName, plans, trialSchema, TrialSchema(includeAdditionalParamsInSchema, true)),
+    OwnClusterPlanID: defaultServicePlan(OwnClusterPlanID, OwnClusterPlanName, plans, trialSchema, TrialSchema(includeAdditionalParamsInSchema, true)),
```

Comment:
we cannot use trial schema which contains one field. We need kubeconfig/shootName/shootDomain there